hono/package.json

```diff
@@ -9,10 +9,12 @@
     "wrdev-migrate": "npm run wrdev-migrate-local && npm run wrdev-migrate-remote",
     "wrdev-migrate-local": "wrangler d1 migrations apply dev-shafa-hono-db --local",
     "wrdev-migrate-remote": "wrangler d1 migrations apply dev-shafa-hono-db",
+    "wrtest-migrate-local": "wrangler d1 migrations apply test-shafa-hono-db --local --env test --persist-to=./test/util/tmp-d1-db",
     "wrstage-migrate": "wrangler d1 migrations apply stage-shafa-hono-db --env stage",
     "wrprod-migrate": "wrangler d1 migrations apply prod-shafa-hono-db --env production",
     "generate": "drizzle-kit generate:sqlite --schema=src/schema.ts",
     "test": "jest test",
+    "db-clean": "wrangler d1 execute test-shafa-hono-db --local --env test --file=./test/util/cleanup-test-db.sql --persist-to=./test/util/tmp-d1-db",
```

Comment:
1. I was thinking that the tmp db directory would follow the path `test/tmp-d1-db`, sorry if that wasn't clear
2. `db-clean` doesn't seem specific enough (which db are we cleaning???), maybe `wrtest-drop-db-local?

Reply:
tmp db directory path has been updated. I think using `clean` is better since we are only removing the contents of the database -  `wrtest-clean-db-local`.

Reply:
Fair, clean does seem better than drop.